### DISCUSSION TITLE
How should this interact with github-pages[bot] (aka 'pages build and deployment')

### DISCUSSION THREAD

**Discussion by davetapley** [JamesIves/github-pages-deploy-action#1073] [Q&A] [answered]
Per [this blog post](https://github.blog/changelog/2021-12-16-github-pages-using-github-actions-for-builds-and-deployments-for-public-repositories/):

> Today you will begin to see a new workflow running called `pages build and deployment` in your public GitHub Pages repositories.

I think there's a lot of scope for confusion for users of this action, when `pages build and deployment` will also be running on every push and pushing pages assets.

Thoughts?

---

See also: https://github.community/t/github-pages-bot-added-to-our-repos-unannounced/218214/8

**Comment by JamesIves** [comment, marked as answer]
I think it's impossible to really say right now what that looks like until GitHub announces what these actions intend to do in the long run. As the post suggests this is a necessary step that occurs after the push gets made, this was already occurring behind the scenes it just wasn't made visible.
